src/AbstractPluginManager.php

```diff
@@ -8,9 +8,8 @@
 
 namespace Laminas\ServiceManager;
 
-use Interop\Container\ContainerInterface;
+use Psr\Container\ContainerInterface;
```

Comment:
Per the previous comment, I think we should keep here `Interop` not `Psr`. We are using `Psr` through the alias, so it should be fine.

Reply:
By having container-interop inlined in the package, we now ensure that anything container-interop is also PSR-11, and since we only consume the PSR-11 interfaces within the plugin manager, I think typehinting against it here is acceptable, and signals the intent for future versions.